### PR TITLE
Added resources for self-organized workshops.

### PR DESCRIPTION
**Pull request by ErinBecker**
@tracykteal @maneesha Please take a look and let me know if you see anything that is inaccurate or unclear. 

Summary of changes: 
- Added checklists and email templates for self-organized workshops (many of these can be repurposed for DC-organized workshops)
- Modified styles to enable checkboxes in lists
- Added link to self-organized workshops page from main workshops page

Suggestions for additions to troubleshooting page would be greatly appreciated. I'm tempted to leave this page out for now because not much info there.

### CODE REVIEW

**Comment by ErinBecker** [comment]
To add to troubleshooting:

- How to handle CoC violations.
- What to do if instructor cancels.
- Others?

File: pages/instructor-checklist.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- [ ] Read through our [troubleshooting document](/troubleshooting/)  
+
+## At the workshop:  
+- [ ] Remind your learners about the [Code of Conduct](http://www.datacarpentry.org/code-of-conduct/)  
+- [ ] Remind learners to use their sticky notes to give feedback - see description of how these are used [here](http://swcarpentry.github.io/instructor-training/16-practices/)  
+- [ ] Get feedback with minute cards at lunch and end of each day - see description of minute cards [here](http://swcarpentry.github.io/instructor-training/16-practices/)
+
+## After the workshop:  
+- [ ] Take part in an [instructor discussion session](http://pad.software-carpentry.org/instructor-discussion)  
+- [ ] Send any feedback you have to [Data Carpentry admin](mailto:admin@datacarpentry.org) (admin@datacarpentry.org)
```
Comment (maneesha) [inline pages/instructor-checklist.md]: "Send attendance counts and any feedback you have ...."

File: pages/instructor-checklist.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- [ ] Report any new [configuration problems](https://github.com/swcarpentry/workshop-template/wiki/Configuration-Problems-and-Solutions)  
+- [ ] Report any lesson errors or suggest improvements on GitHub or by [email](mailto:admin@datacarpentry.org) (admin@datacarpentry.org) 
+
```
Comment (maneesha) [inline pages/instructor-checklist.md]: Link to lessons on github here

File: pages/instructor-checklist.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+header:
+   image_fullwidth: "wood_plank.jpg"
+permalink: "/instructor-checklist/"
+---
+
+Each instructor is responsible for preparing to teach their section(s) of the workshop. Remember that our lessons are continuously evolving, so even if you have taught the content before, you will need to check the materials for changes.
+
+## Before the workshop:  
+- [ ] Coordinate with the lead instructor to decide who is teaching what, and in which order  
+- [ ] If your workshop is self-organized, meet with the lead instructor to be trained in [Data Carpentry’s teaching practices](https://swcarpentry.github.io/instructor-training/16-practices/)
```
Comment (maneesha) [inline pages/instructor-checklist.md]: "If your workshop is self-organized, and you are not a certified Data Carpentry instructor, meet with the lead instructor to be trained...."

**Comment by maneesha** [comment]
Add to troubleshooting what to do if 

- projector fails
- internet goes out
- packages can not be installed on learner computer (they don't have admin access, etc.)

File: pages/instructor-checklist.md (hunk 1 above)
Comment (ErinBecker) [inline pages/instructor-checklist.md]: I assume lead instructor is responsible for sending attendance counts, so this instruction is currently only on the lead instructor checklist.

File: pages/instructor-checklist.md (hunk 2 above)
Comment (ErinBecker) [inline pages/instructor-checklist.md]: With current organization of lessons, linking to lessons directly on github would be 12 different links. Will add link to lesson homepage, which has github repo links on it.

**Pull request merged**